bindings/matrix-sdk-ffi/src/timeline.rs

```diff
@@ -56,18 +56,30 @@ pub fn message_event_content_new(
 #[uniffi::export]
 pub fn message_event_content_from_markdown(
     md: String,
+    emote: bool,
 ) -> Arc<RoomMessageEventContentWithoutRelation> {
-    Arc::new(RoomMessageEventContentWithoutRelation::new(RumaMessageType::text_markdown(md)))
+    if emote {
+        Arc::new(RoomMessageEventContentWithoutRelation::new(RumaMessageType::emote_markdown(md)))
+    } else {
+        Arc::new(RoomMessageEventContentWithoutRelation::new(RumaMessageType::text_markdown(md)))
+    }
```

Comment:
nit (here and below): in Rust, if statements actually are expressions, so you can shorten the code to:

```suggestion
    Arc::new(RoomMessageEventContentWithoutRelation::new(if is_emote { ... } else { ... })
```